include/nbl/asset/utils/CSPIRVIntrospector.h

```diff
@@ -13,172 +17,657 @@
 #include "nbl/asset/ICPUImageView.h"
 #include "nbl/asset/ICPUComputePipeline.h"
 #include "nbl/asset/ICPURenderpassIndependentPipeline.h"
-#include "nbl/asset/utils/ShaderRes.h"
+//#include "nbl/asset/utils/ShaderRes.h"
```

Comment:
does that file even exist anymore/is useful for anything?

Reply:
it does exist, idk if it is used anywhere

Reply:
remove it and see

Reply:
Nabla does compile without it, deleting